playwright/tests/slices/slicesList.spec.ts

```diff
@@ -64,19 +65,43 @@ test.describe("Slices", () => {
       await slicesListPage.clickCard(newSliceName);
 
       await expect(
-        sliceBuilderPage.breadcrumb.getByText(newSliceName),
+        sliceBuilderPage.getBreadcrumbLabel(newSliceName),
       ).toBeVisible();
     },
   );
 
+  // See: #791
```

Comment:
What does #791 refer to? Maybe we can add a URL here to make it extra clear.

```ts
// See: https://github.com/prismicio/slice-machine/issues/791
```

Reply:
Agree, since we have both Linear and GitHub, yes it refers to GitHub